CHANGELOG_PENDING.md

```diff
@@ -1,6 +1,12 @@
 ### SDK Features
 
 ### SDK Enhancements
-
+* `aws`: Provides more customization options for retryer by adding a constructor for default Retryer which accepts functional options. Adds NoOpRetryer to support no retry behavior. Exposes members of default retryer. [#375](https://github.com/aws/aws-sdk-go-v2/pull/375)
+    * Updates the underlying logic used by the default retryer to calculate jittered delay for retry. 
+    * Handles int overflow for retry delay. Fixes [#370](https://github.com/aws/aws-sdk-go-v2/issues/370)
```

Comment:
Suggest updating this so that their is a short title for the entry, with a paragraph for the description.